src/myr/Myr.js

```diff
@@ -289,17 +289,39 @@ class Myr {
   makeDroppable = (outerElId, mass = 2) => {
     let el = this.getEl(outerElId);
     let dynamicBody = `shape: auto; mass: ${mass}; angularDamping: 0.5; linearDamping: 0.5;`;
-    el["dynamic-body"] = dynamicBody;
+    el["dynamic-body"] = dynamicBody;   
     return outerElId;
   }
 
   // Disallows the entity to be dropped
   makeUnDroppable = (outerElId, mass = 2) => {
     let el = this.getEl(outerElId);
-    el["dynamic-body"] = "";
+    //Only makes an item undroppable if it is droppable but is not pushable
+    if(el["dynamic-body"] && (!el["force-pushable"] || el["force-pushable"] === "false")) {
+      el["dynamic-body"] = null;
+    }
     return outerElId;
   }
 
+  // Allows the entity to be pushed
```

Comment:
Was this run through the formatter? There is an option to auto-format on save that is best if it is turned on.